src/e-load-xcode.js

```diff
@@ -48,6 +51,23 @@ for (const sdk of SDK_TO_LINK) {
   childProcess.execFileSync('sudo', ['ln', '-s', sourceSDK, targetDirectory]);
 }
 
+// Unlink unnecessary macOS SDKs.
+for (const sdk of SDK_TO_UNLINK) {
+  const sourceSDK = path.resolve(macOSSDKs.path, `MacOSX${sdk}.sdk`);
+  if (!fs.existsSync(sourceSDK)) continue;
+
+  const targetDirectory = path.resolve(xCodeSDKDir, `MacOSX${sdk}.sdk`);
+  if (!fs.existsSync(targetDirectory)) continue;
+
+  console.warn(
+    `${color.info} Removing symbolic link from ${color.path(sourceSDK)} --> ${color.path(
+      targetDirectory,
+    )}`,
+  );
+
+  childProcess.execFileSync('sudo', ['unlink', sourceSDK]);
```

Comment:
I've just realized I don't think we need `sudo` for this any more but I'll check in a separate PR

Reply:
Yeah we don't need `sudo` for this https://github.com/electron/build-tools/pull/112/files